modules/dists/ReplicatedDist.chpl

```diff
@@ -424,6 +424,16 @@ class ReplicatedArr : BaseArr {
         halt("locale ", here.id, " has no local replicand");
     return localArrs[here.id];
   }
+
+  /* Access another locale's local array representation by locale id */
+  proc replicand(id: here.id.type) ref {
```

Comment:
It seems to me as though this interface would be more natural if it took a locale rather than a locale ID.  Was there a reason it needed to be the latter?

(This isn't something that would normally occur to me, but @mppf was asserting this morning that he believes well-chosen method names should be verb-like rather than nouns in order to make code read like English, which might suggest calling this `getReplicand()` rather than simply `replicand()` if you agree with that philosophy.  That said, I wouldn't have thought of this if we hadn't had the conversation this morning (and I realize that I'm the one who probably suggested we call this `replicand` in the first place)).

Completely unrelated to this PR, I"m noting that `ref` doesn't get highlighted above.  :'(